### PR TITLE
Externalize preview status doc in guides

### PR DESCRIPTION
**Pull request by emmanuelbernard**
Also add experimental status

### CODE REVIEW

**Comment by emmanuelbernard** [comment]
Also stable guides do not include the status adoc file to not add weight to the text.

**Comment by emmanuelbernard** [comment]
for #issue 6042

File: docs/src/main/asciidoc/status-include.adoc [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ifeval::["{extension-status}" == "preview"]
+In _preview_, backward compatibility and presence in the ecosystem is not guaranteed.
+Specific improvements might require to change configuration or APIs and plans to become _stable_ are under way.
+Feedback is welcome on our https://groups.google.com/d/forum/quarkus-dev[mailing list] or as issues in our https://github.com/quarkusio/quarkus/issues[GitHub issue tracker].
+endif::[]
+ifeval::["{extension-status}" == "stable"]
+Being _stable_, backward compatibility and presence in the ecosystem are taken very seriously.
+endif::[]
+
+For a full list of possible extension statuses, check our https://some.url/faq#status[FAQ entry].
```
Comment (gsmet) [inline docs/src/main/asciidoc/status-include.adoc]: I fixed the URL here to point to https://quarkus.io/faq/#extension-status but we will need to add the content there.

**Review by gsmet — APPROVED** [review]

File: docs/src/main/asciidoc/status-include.adoc (hunk 1 above)
Comment (emmanuelbernard) [reply docs/src/main/asciidoc/status-include.adoc]: FAQ created

**Pull request merged**